src/test/java/com/fasterxml/jackson/databind/ser/jdk/SqlDateSerializationTest.java

```diff
@@ -84,9 +85,13 @@ public void testSqlTimestamp() throws IOException
     
     public void testPatternWithSqlDate() throws Exception
     {
+        ObjectMapper mapper = new ObjectMapper();
+        // `java.sql.Date` applies system default zone (and not UTC)
+        mapper.setTimeZone(TimeZone.getDefault());
```

Comment:
Ah. This also suggests that handling might be wrong (wrt all other date/time datatypes). But makes sense regarding stability.